packages/tools/examples/segmentBidirectionalTool/index.ts

```diff
@@ -0,0 +1,435 @@
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  setVolumesForViewports,
+  volumeLoader,
+  cache,
+} from '@cornerstonejs/core';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  addDropdownToToolbar,
+  addSliderToToolbar,
+  setCtTransferFunctionForVolumeActor,
+  addButtonToToolbar,
+} from '../../../../utils/demo/helpers';
+import createBidirectionalToolData from './createBidirectionalToolData';
+import contourAndFindLargestBidirectional from './findLargestBidirectional';
```

Comment:
this is odd importing, import A from './B' but I prefer this actually to be inside the tools/utilities/measuremnts or something